editor/import/resource_importer_csv_translation.cpp

```diff
@@ -131,7 +132,24 @@ Error ResourceImporterCSVTranslation::import(const String &p_source_file, const
 			xlt = cxl;
 		}
 
-		String save_path = p_source_file.get_basename() + "." + translations[i]->get_locale() + ".translation";
+		// `.right(6)` strips away the leading "res://".
+		String stripped_path = TranslationServer::TRANSLATION_FILES_PATH.plus_file(p_source_file.get_basename().right(6));
```

Comment:
This might not work anymore, `right()` was changed in #36180. This should use the clearer `substr()`.

Reply:
I updated the PR to use `substr`. I also tried testing it again, and I wasn't able to get the translation demo to work either with or without this PR. For now I will assume that this specific code still works because it worked when I made the PR.